### PR TITLE
Minor bugfixes

### PR DESCRIPTION
**Pull request by mrow4a**
- update documentation

- properly handle error when creating file

Zrzut ekranu 2020-11-16 o 21 02 31

- fixes https://github.com/owncloud/appliance/issues/94 -> remove default wopi_url and use hint instead (important for appliance installation @voroyam )

Zrzut ekranu 2020-11-16 o 21 29 15

### CODE REVIEW

**Review by karakayasemi — APPROVED** [review]
Looks good to me.

File: lib/AppConfig.php [hunk 1]
```diff
@@ -20,7 +20,6 @@
 class AppConfig {
 	private $appName = 'richdocuments';
 	private $defaults = [
-		'wopi_url' => 'https://localhost:9980',
```
Comment (jvillafanez) [inline lib/AppConfig.php:23]: I'm not sure about the consequences of this change... Taking into account that it's expected the admin to adjust the url anyway, I'd prefer to keep it.

Comment (voroyam) [reply lib/AppConfig.php:23]: If this is irrelevant - I am for leaving it out.

For the appliance we need this to be empty on start to so we can set the appropriate settings for onlyoffice and collabora integration. A default value prevent those automatic settings. That's why I requested to removal of this default value.

Comment (mrow4a) [reply lib/AppConfig.php:23]: @jvillafanez I deep analyzed, no consequence

**Review by jvillafanez — APPROVED** [review]

**Comment by mrow4a** [comment]
@phil-davis can you merge?

**Comment by phil-davis** [comment]
codecov failed. I do not have privs to merge.
@micbar maybe?

**Pull request merged**